src/frontend/src/views/Data/components/GalagoConfirmationModal/index.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  AnalyticsTreeViewNextstrain,
+  EVENT_TYPES,
+} from "src/common/analytics/eventTypes";
+import { analyticsTrackEvent } from "src/common/analytics/methods";
+import { RedirectConfirmationModal } from "src/common/components/library/data_subview/components/RedirectConfirmationModal";
+import galagoLogo from "src/common/images/galago-logo-beta.png";
+// TODO: (ehoops) - This button will change when we create the galago URL in sc-214181
+// currently this is just using the nextstrain url as a placeholder
+import { ConfirmButton } from "src/common/utils/TreeModal/ConfirmButton";
+
+interface Props {
+  open: boolean;
+  onClose: () => void;
+  treeId: number;
+}
+
+const NextstrainConfirmationModal = ({
+  open,
+  onClose,
+  treeId,
+}: Props): JSX.Element => {
+  const content = (
+    <>
+      By clicking “Continue” you agree to send a copy of your tree JSON to
+      Galago (Beta), a separate, but related service from CZ GEN EPI. Galago is
+      a serverless web application which runs entirely in the browser. Galago
+      does not store or share your data; however, you may choose to share the
+      URL with others.{" "}
```

Comment:
What's the final forced space `{" "}` getting us here? I'm used to it when there's multiple text components that get spaces stripped between them, but since this is a free-floating paragraph, I don't see what the space is preventing adjacency on.

Reply:
This was just some copy pasta since the NextStrain modal has a Learn More link there.